### ISSUE TITLE
[Pontoon] Language.DoesNotExist (underscore vs dash in language settings)

### ISSUE THREAD

**Issue by patjouk** [mozilla/donate-wagtail#556]
From Sentry:

```
Language.DoesNotExist: Language matching query does not exist.
(8 additional frame(s) were not displayed)
...
  File "wagtail_localize_pontoon/importer.py", line 151, in import_file
    resource, language = PontoonResource.get_by_po_filename(filename)
  File "wagtail_localize_pontoon/models.py", line 113, in get_by_po_filename
    Language.get_by_rfc5646_language_tag(parts[1]),
  File "wagtail_localize/models.py", line 81, in get_by_rfc5646_language_tag
    return cls.objects.get(code=language_tag.lower())
  File "django/db/models/manager.py", line 82, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "django/db/models/query.py", line 408, in get
    self.model._meta.object_name
```

It seems to be linked to Pontoon generating `"Language: gu_IN\n"` while Wagtail expects `gu-IN`.

**Comment by kaedroho** [comment]
This bit of code is triggered when Wagtail detects a file changed in the repo, if the filename matches a pattern, it'll use the filename to look for the locale. The files are originally created and named by Wagtail so these should always exist unless the languages were deleted from Wagtail, the Wagtail site was rebuilt, or more than one Wagtail site is using the same repo.

**Comment by TheoChevalier** [comment]
This is the commit that triggered the error: https://github.com/mozilla-l10n/mozilla-donate-content/commit/e983697682801a209ad524a85382e3d0589fb876

Is gettext involved at all in the process?

**Comment by kaedroho** [comment]
> Is gettext involved at all in the process?

Nope, these PO files are all generated from content in the database and formatted using `polib`

**Comment by TheoChevalier** [comment]
ok. And the repo is used by a single instance (give.mozilla.org)

**Comment by kaedroho** [comment]
Ahh, I think the original message in this issue is accurate, `wagtail-localize-pontoon` assumes that your `LANGAUGES` setting has all lowercase language codes: https://docs.djangoproject.com/en/2.2/topics/i18n/

Looks like we will have to alter the `PontoonResource.get_by_po_filename` function to use a different version of `get_by_rfc5646_language_tag` that works with the uppercase region format.

**Comment by kaedroho** [comment]
Just released `wagtail-localize-pontoon` 0.1.3 with a fix for this issue

**Comment by TheoChevalier** [comment]
Confirming this is fixed! https://donate-wagtail.mofostaging.net/gu-IN/
